platform-api/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceService.java

```diff
@@ -392,35 +387,12 @@ private UsersWorkspaceDto asUsersWorkspaceDto(UsersWorkspace workspace) {
                                            .build(workspace.getId())
                                            .toString(),
                                  APPLICATION_JSON,
-                                 GET_RUNTIME_WORKSPACE));
+                                 LINK_REL_GET_RUNTIMEWORKSPACE));
         }
-
-        final List<CommandDto> commands = workspace.getCommands()
-                                                   .stream()
-                                                   .map(this::asDto)
-                                                   .collect(toList());
-        final List<ProjectConfigDto> projects = workspace.getProjects()
-                                                         .stream()
-                                                         .map(this::asDto)
-                                                         .collect(toList());
-        final Map<String, EnvironmentDto> environments = workspace.getEnvironments()
-                                                                  .values()
-                                                                  .stream()
-                                                                  .collect(toMap(Environment::getName, this::asDto));
-
-        return newDto(UsersWorkspaceDto.class).withId(workspace.getId())
-                                              .withStatus(workspace.getStatus())
-                                              .withName(workspace.getName())
-                                              .withOwner(workspace.getOwner())
-                                              .withDefaultEnvName(workspace.getDefaultEnvName())
-                                              .withCommands(commands)
-                                              .withProjects(projects)
-                                              .withEnvironments(environments)
-                                              .withAttributes(workspace.getAttributes())
-                                              .withLinks(links);
+        return workspace.withLinks(links);
     }
 
-    private RuntimeWorkspaceDto asDto(RuntimeWorkspace workspace) {
+    private RuntimeWorkspaceDto asRuntimeWorkspaceDto(RuntimeWorkspace workspace) {
```

Comment:
The same [comment](https://github.com/codenvy/che-core/pull/307#discussion_r41075354)